improver_tests/acceptance/test_wxcode.py

```diff
@@ -44,14 +44,15 @@ def test_basic(tmp_path):
     """Test basic wxcode processing"""
     kgo_dir = acc.kgo_root() / "wxcode/basic"
     kgo_path = kgo_dir / "kgo.nc"
-    params = ["lightning_flashes_per_unit_area_in_vicinity_above",
-              "cloud_area_fraction_above",
-              "rainfall_rate_above",
-              "rainfall_rate_in_vicinity_above",
+    params = ["cloud_area_fraction_above",
```

Comment:
Looks like this list is the same for at least two of the tests, and only missing one element for a third.  Would it be better to define it at the top of the module?

Reply:
Refactored.